pkgs/tools/security/cdxgen/default.nix

```diff
@@ -1,26 +1,66 @@
-{ lib
-, fetchFromGitHub
-, buildNpmPackage
+{ fetchFromGitHub
+, lib
+, makeWrapper
+, nodejs
+, nodePackages
+, pnpm
+, python3
+, stdenv
+, xcbuild
 }:
 
-buildNpmPackage rec {
+stdenv.mkDerivation rec {
```

Comment:
I am not sure if something changed recently, but last time I checked, we were using the `finalAttrs` pattern for `stdenv.mkDerivation`.